sdk/elixir/lib/dagger.ex

```diff
@@ -43,7 +43,18 @@ defmodule Dagger do
   @doc """
   Disconnecting Dagger.
   """
-  def close(%Dagger.Client{client: client}) do
-    Dagger.Internal.Client.close(client)
+  def close(%Dagger.Client{client: inner} = client) do
+    case Dagger.Client.stop(client, timeout: 10) do
```

Comment:
The timeout is quite weird for me when first read it. Because in Erlang/Elixir, they use milliseconds. In my head thought, why timeout is super quick? 😂

Reply:
That means we need timeout at the request scope right? If that is so, I think it would be easy but need to bypass timeout to the HTTP client that uses it internally.

Reply:
Yeah, timing out the request would be sufficient. It's technically different from now, where the server implements a timeout internal to the request and will complete the request, but I think it'd be functionally equivalent for all intents and purposes.

Reply:
@wingyplus Just following up before I mark this resolved in case you started on anything here: SDKs no longer need to be updated. :tada: